src/devices/bus/abcbus/cadmouse.h

```diff
@@ -0,0 +1,43 @@
+// license:BSD-3-Clause
+// copyright-holders:Peter Bortas
+#ifndef MAME_BUS_ABCBUS_CADMOUSE_H
+#define MAME_BUS_ABCBUS_CADMOUSE_H
+
+#pragma once
+
+#include "abcbus.h"
+#include "cpu/z80/z80.h"
+
+//**************************************************************************
+//  TYPE DEFINITIONS
+//**************************************************************************
+
+class abc_cadmouse_device :  public device_t,
+	public device_abcbus_card_interface
+{
+public:
+	// construction/destruction
+	abc_cadmouse_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+
+	void abc_cadmouse_mem(address_map &map);
+	void abc_cadmouse_io(address_map &map);
```

Comment:
It should be possible to make these internal memory maps protected now.

Reply:
Ok, I'll move them